webpack.prod.js

```diff
@@ -0,0 +1,67 @@
+const path = require('path');
```

Comment:
It's great that we have separate configs for dev and prod now! I see a little bit of duplication like with the output config. We could have a base config file where we put any values that are the same for both configs. There's little duplication right now so no big deal, but if you see this changing in the future it might be a good thing to start now.

Reply:
Agreed.  Working on splitting into dev, prod, and common now.